skxray/core.py

```diff
@@ -1180,3 +1180,44 @@ def multi_tau_lags(multitau_levels, multitau_channels):
     lag_steps = np.append(lag_steps, np.array(lag))
 
     return tot_channels, lag_steps
+
+
+def geometric_bin_edges(common_ratio, number_of_images, first_term=1):
```

Comment:
geometric_bin_edges, use case is for XSVS analysis, I will put new PR soon for that